updatecli/.ci/scripts/release.bash

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Ensure we work from the updatecli directory
+# This is even more important as we use the policy path to generate the policy reference
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+pushd "$SCRIPT_DIR/../../"
+
+: "${UPDATECLI_GITHUB_OWNER:=kubewarden}" # useful for local execution
+
+: "${SOURCE_REPOSITORY:=https://github.com/kubewarden/automation/updatecli}"
+: "${SOURCE_REPOSITORY_FOLDER:=https://github.com/kubewarden/automation/tree/main/updatecli}"
+: "${POLICIES_ROOT_DIR:=policies}"
+: "${POLICY_ERROR:=false}"
+: "${OCI_REPOSITORY:=ghcr.io/${UPDATECLI_GITHUB_OWNER}/updatecli}"
```

Comment:
```suggestion
: "${OCI_REPOSITORY:=ghcr.io/${UPDATECLI_GITHUB_OWNER}/automation}"
```

Reply:
Went with `ghcr.io/kubewarden/automation/updatecli` on 0ea0b5c, resolving the other conversations.